src/common/menu-utils/index.ts

```diff
@@ -26,18 +26,18 @@ export class MenuUtils<
     State extends MenuState,
 > extends Component<Input, State> {
     declare type?: string;
-    declare items: MenuItem[];
+    declare item: MenuItem[];
```

Comment:
Technically this one can be plural, since it's internal. Maybe `items` or `itemList` is better